### PR TITLE
Shuttle movement tweaks

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Shuttles now check that the turf they are moving is not the baseturf. If its the baseturf, it wont move/copy that turf.

In simpler terms, if you poke a hole to the shuttle (and make space, which has a baseturf of space) that tile wont "move" with the shuttle/replace what is at the other end.

So if you blow up half the escape shuttle, and fly it to lavaland it wont drag space with it.

More immediately this means the escape shuttle will break to transit space instead of carrying around blocks of still space, making destroyed shuttles extra dangerous to be on.

Objects on said broken spots wont move either

Tested this with the emergency shuttle and unless a mapper did somethinf terrible it should work with all shuttles.

:cl: Kor
rscadd: Shuttles will no longer drag random tiles of space with them. This means you can be thrown out of partially destroyed escape shuttles very easily.
rscadd: The people and objects on destroyed tiles won't be along for the ride when the shuttle moves either. 
/:cl:

### CODE REVIEW

**Comment by Bawhoppen** [comment]
very good

**Comment by optimumtact** [comment]
awesome, I noticed this a while ago and it was ugly looking!

**Comment by Incoming5643** [comment]
at some point in the past this used to happen, but the effect was lost when the shuttle code changed

:+1:

**Comment by Shapsy** [comment]
Oh boy, get hyped for more last second minibombs
